pom.xml

```diff
@@ -93,9 +110,14 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.18.1</version>
                 <configuration>
-                    <testSourceDirectory>${basedir}/src/main/java/</testSourceDirectory>
-                    <testClassesDirectory>${project.build.directory}/classes/</testClassesDirectory>
```

Comment:
This was a side effect from when the library and the sample app were in the same project. The library doesn't need this anymore but I think the sample app will. Anyway, a valid change. Just thought would mention this here for historical documentation.
/cc @phillbarber